src/runners/rfqm_worker_runner.ts

```diff
@@ -94,7 +97,10 @@ export async function runRfqmWorkerAsync(rfqmService: RfqmService, workerAddress
     });
 
     // Start the consumer - aka the worker
-    consumer.consumeAsync().catch((e) => logger.error(e));
+    consumer.consumeAsync().catch((e) => {
+        logger.error({ error: e }, 'Unexpected error encountered in consume loop');
+        process.exit(1);
```

Comment:
what is the error is a retryable error?

Reply:
shouldn't ever reach this line. At this point, the only errors that we experience would be critical and unexpected